ldm/models/diffusion/shared_invokeai_diffusion.py

```diff
@@ -294,25 +298,29 @@ def _threshold(self, threshold, warmup, latents: torch.Tensor, sigma) -> torch.T
         if self.debug_thresholding:
             std, mean = [i.item() for i in torch.std_mean(latents)]
             outside = torch.count_nonzero((latents < -current_threshold) | (latents > current_threshold))
-            print(f"\nThreshold: 𝜎={sigma.item()} threshold={current_threshold:.3f} (of {threshold:.3f})\n"
+            print(f"\nThreshold: %={percent_through} threshold={current_threshold:.3f} (of {threshold:.3f})\n"
                   f"  | min, mean, max = {minval:.3f}, {mean:.3f}, {maxval:.3f}\tstd={std}\n"
                   f"  | {outside / latents.numel() * 100:.2f}% values outside threshold")
 
         if maxval < current_threshold and minval > -current_threshold:
             return latents
 
         if maxval > current_threshold:
+            latents = torch.clone(latents)
```

Comment:
why clone?

Reply:
I got a fatal error when I didn't copy the latents. This may not be the way to resolve it; I still don't know much about working with torch's tensors.